index.html

```diff
@@ -0,0 +1,611 @@
+            <span class="icon-computer"></span>
+            <h2>Responsive design</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-lab"></span>
+            <h2>Easy to customize</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-airplane"></span>
+            <h2>Perfect showcase</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+          <li class="flag">
+            <span class="icon-cog"></span>
+            <h2>App store support</h2>
+            <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+          </li>
+        </ul>
+      </section>
+      <section data-section="conference-support">
+        <div class="conference-text-wrapper">
+          <p class="btn-small">New</p>
+          <h1>Live conference support available</h1>
+          <p>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes.</p>
+          <aside>
+            <h2>Free Download On</h2>
+            <div>
+              <a target="_blank" href="https://play.google.com/store/apps?hl=en">
+                <span class="icon-android"></span><br>
+                <p>Android</p>
+              </a>
+            </div>
+            <div>
+              <a target="_blank" href="https://itunes.apple.com/us/genre/ios/id36?mt=8&cid=wwa-us-kwg-features">
+                <span class="icon-apple2"></span><br>
+                <p>iOS</p>
+              </a>
+            </div>
+          </aside>
+        </div>
+        <img src="/img/portrait-display-mockup.png" alt="" title="">
+      </section>
+      <section data-section="why-appbox" id="why">
+        <h1 class="header-underline">Why Choose AppBox</h1>
+        <img src="/img/iphone-6.png" alt="Image of iPhone" title="iPhone">
+        <ul class="why-1">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-cubes"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>App store</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+          </li>
+          <img src="/img/spacer-dots.svg">
+          <li>
+            <div class="why-icon-wrapper">
+              <span class="icon-screen"></span>
+            </div>
+            <div class="why-item-wrapper">
+              <h2>Retina ready</h2>
+              <p>Eleifend morbi velit porttitor sed ipsum imperdiet ac ullamcorper fusce</p>
+            </div>
+          </li>
+          <img src="/img/spacer-dots.svg">
```

Comment:
Be sure to run your markup through a validator: https://validator.w3.org/nu/?doc=http%3A%2F%2Futopian-whistle.surge.sh%2F

`<img>` tags need to close themselves; this should be: `<img src="/img/spacer-dots.svg" />`